### PR TITLE
Add projects about me

### PR DESCRIPTION
**Pull request by GreetyCr**
Add section about me (projects)
Add section works
Add background of works (.svg)
Use DRY in CSS as much as possible

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,5 +58,116 @@ <h2 class="about-me__info--sub-title">I'm a Software Developer</h2>
         </li>
       </ul>
     </main>
+    <section class="projects__container ">
```
Comment (DuaneDave) [inline index.html:61]: According to the requirements, this container should be aligned using grid. kindly implement this to make the styling easier across devices and to stick with the project requirements.

File: index.html [hunk 2]
```diff
@@ -58,5 +58,116 @@ <h2 class="about-me__info--sub-title">I'm a Software Developer</h2>
...
+          <ul class="flex_row">
+            <li class="project__card--lenguage flex_column">CSS</li>
+            <li class="project__card--lenguage flex_column">HTML</li>
+            <li class="project__card--lenguage flex_column">Bootstrap</li>
+            <li class="project__card--lenguage flex_column">Ruby</li>
+          </ul>
+        </div>
+        <button class="project__card--button project__card--button--position ">
+          See Project
+        </button>
```
Comment (DuaneDave) [inline index.html:85]: As a requirement for the buttons in this section, they are expected to have multiple state such as hover, active/pressed and so on in order to make them very interactive. In line with this, I recommend styling the buttons accordingly.

Use this color schemes
btn

This applies to all buttons

File: index.html [hunk 3]
```diff
@@ -58,5 +58,116 @@ <h2 class="about-me__info--sub-title">I'm a Software Developer</h2>
...
+            <li class="project__card--lenguage project__card--lenguage-secondary flex_column white">HTML</li>
+            <li class="project__card--lenguage project__card--lenguage-secondary flex_column white">Bootstrap</li>
+            <li class="project__card--lenguage project__card--lenguage-secondary flex_column white">Ruby</li>
+          </ul>
+        </section>
+        <button class="project__card--button project__card--button-secondary">
+          See Project
+        </button>
+      </article>
+    </section>
```
Comment (DuaneDave) [inline index.html:171]: I notice you didn't work on the about section, consider working on it according to [specifications](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=the%20%22About%20myself%22%20section%20(placed%20after%20the%20projects%20and%20before%20the%20contact%20form)%3A) to meet up with the requirements.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION> 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔️ Nice styling and organization of the work cards
✔️ Very descriptive PR summary


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -58,5 +58,127 @@ <h2 class="about-me__info--sub-title">I'm a Software Developer</h2>
...
+      <h1 class="about-me__info--title">
+        About me
+      </h1>
+      <p class="about-me__info--para--secondary">
+        Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+      </p>
+      <button class="project__card--button project__card--button--position ">
+        Get my resume
+      </button>
+    </section>
```
Comment (binhussen) [inline index.html:182]: - Great job for implementing the About section as the previous code reviewer suggested, however, you didn’t add the whole part please add it as it is part of the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) 👌 

#### Please Add the remaining one 👇

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @GreetyCr , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -58,5 +58,265 @@ <h2 class="about-me__info--sub-title">I'm a Software Developer</h2>
...
+          ><span class="tech-stack__list--title"> Frameworks</span>
+        </li>
+        <ul class="tech-stack__items">
+          <li class="tech-stack__item">React.js</li>
+          <li class="tech-stack__item">Ruby on Rails</li>
+          <li class="tech-stack__item">RSpec</li>
+          <li class="tech-stack__item">Capybara</li>
+        </ul>
+        <li class="tech-stack__title">
+          <i class="fa-solid fa-square"></i
```
Comment (franclobo) [inline index.html:309]: - [ ] The image before Skills is a circle, please change it to be similar to the Figma design 👇🏽 

| Figma design | Your design |
|---|---|
| image | image |

**Review by franclobo — APPROVED** [review]
Hi @GreetyCr, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 
- Professional README file ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**